src/Domain/Campaign.php

```diff
@@ -134,14 +146,16 @@ class Campaign extends SalesforceReadProxy
 
 
     /**
-     * @param \DateTimeImmutable|null $regularGivingCollectionEnd
      * @param Salesforce18Id<Campaign> $sfId
+     * @param Salesforce18Id<MetaCampaign>|null $metaCampaignSfId
```

Comment:
good use case for the generic phantom type here - SA tools will make sure don't get these two mixed up even though they've got the same type at runtime.